packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -117,7 +124,17 @@ const CFDFinancialStpRealAccountSignup = (props: TCFDFinancialStpRealAccountSign
         }
         return need_poi_for_bvi_labuan;
     };
-    const should_show_poa = !['pending', 'verified'].includes(authentication_status.document_status);
+
+    let should_show_poa = false;
+
+    if (
+        jurisdiction_selected_shortcode === Jurisdiction.LABUAN &&
+        (account_status.risk_classification === 'high' || poa_resubmit_for_labuan)
```

Comment:
It is the same. Just cerated a variable because any enhancements in future for POA resubmission can be just modified in getAuthenticationStatusInfo and the status gets reflected everywhere